static/app/views/releases/thresholdsList/thresholdGroupRows.tsx

```diff
@@ -6,92 +6,86 @@ import moment from 'moment';
 import {APIRequestMethod} from 'sentry/api';
 import {Button} from 'sentry/components/button';
 import {CompactSelect} from 'sentry/components/compactSelect';
-import ProjectBadge from 'sentry/components/idBadge/projectBadge';
 import Input from 'sentry/components/input';
 import {IconAdd, IconClose, IconDelete, IconEdit} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
-import {Environment, Project} from 'sentry/types';
+import {Project} from 'sentry/types';
 import {getExactDuration, parseLargestSuffix} from 'sentry/utils/formatters';
 import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
 
-import {NEW_GROUP_PREFIX, NEW_THRESHOLD_PREFIX} from '../utils/constants';
-import {EditingThreshold, NewThresholdGroup, Threshold} from '../utils/types';
+import {NEW_THRESHOLD_PREFIX} from '../utils/constants';
+import {EditingThreshold, Threshold} from '../utils/types';
 
 type Props = {
-  orgSlug: string;
+  allEnvironmentNames: string[];
+  isLastRow: boolean;
+  project: Project;
   refetch: () => void;
-  setError: (msg: string) => void;
-  newGroup?: NewThresholdGroup;
+  setTempError: (msg: string) => void;
```

Comment:
How is error different from temp error?

Reply:
The temp error has a timeout to remove itself